### PR TITLE
Template/delivery

### PR DESCRIPTION
**Pull request by Bianca91**
Still need to do the following:
- change the date, bon nr and order TextFields to Card ( user cannot change these fields)
- Need to ensure that paper size A5
I will do the aforesaid changes. Let me know if there is something else. 
Thanks :)
screenshot from 2018-04-11 11-09-06

### CODE REVIEW

**Comment by Ju1111** [comment]
screeeenshots of designs pleaaase <3

**Comment by Bianca91** [comment]
Uploaded a screenshot :-)

File: src/containers/forgotPassword.js [hunk 1]
```diff
@@ -1,56 +1,59 @@
...
 
 //MaterialUI
-import Paper from 'material-ui/Paper'
+import Paper from "material-ui/Paper";
 
 class ForgotPassword extends PureComponent {
-
-  handleSubmit = (data) => {
-    this.props.forgotPassword(data.email)
+  handleSubmit = data => {
```
Comment (Ju1111) [inline src/containers/forgotPassword.js]: why? 🤔

File: src/containers/forgotPassword.js [hunk 2]
```diff
@@ -1,56 +1,59 @@
...
-
-			</div>
-		)
+      </div>
+    );
   }
 }
 
-export default connect(null, { forgotPassword })(ForgotPassword)
+export default connect(null, { forgotPassword })(ForgotPassword);
```
Comment (Ju1111) [inline src/containers/forgotPassword.js]: why is my file redesiiiiigned??

**Pull request closed without merging**